app/vmalert/web.go

```diff
@@ -35,11 +35,11 @@ var (
 		{"-/reload", "reload configuration"},
 	}
 	navItems = []tpl.NavItem{
-		{Name: "vmalert", Url: "../vmalert"},
-		{Name: "Groups", Url: "groups"},
-		{Name: "Alerts", Url: "alerts"},
-		{Name: "Notifiers", Url: "notifiers"},
-		{Name: "Docs", Url: "https://docs.victoriametrics.com/vmalert/"},
+		{Name: "vmalert", URL: "../vmalert"},
+		{Name: "Groups", URL: "groups"},
+		{Name: "Alerts", URL: "alerts"},
+		{Name: "Notifiers", URL: "notifiers"},
```

Comment:
Notifiers are now showed always collapsed. Can we switch them back to visible by default?
<img width="393" alt="image" src="https://github.com/user-attachments/assets/0c17d4d3-686a-487d-9c0e-605f3da22105" />

Reply:
updated